pkg/cluster/px/px.go

```diff
@@ -200,12 +200,12 @@ func (ops *pxClusterOps) Upgrade(new *apiv1alpha1.Cluster, opts *UpgradeOptions)
 	newOCIMonVer := fmt.Sprintf("%s:%s", new.Spec.OCIMonImage, new.Spec.OCIMonTag)
 	newPXVer := fmt.Sprintf("%s:%s", new.Spec.PXImage, new.Spec.PXTag)
 
-	isMajorVerUpgrade, err := ops.isMajorVersionUpgrade(new)
+	isAppDrainNeeded, err := ops.doesUpgradeNeedAppDrain(new)
```

Comment:
The `new` is a reserved keyword in Golang, please do not use variables with the same name (shadowing)
Suggest you rename "new" variable into "newCluster" or similar.

Reply:
fixed